packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -641,11 +641,22 @@ class WebpackCLI {
                     process.exit(2);
                 }
             } else {
-                logger.raw(`${stats.toString(foundStats)}`);
+                const printedStats = stats.toString(foundStats);
```

Comment:
When we use `stats: 'none'` we have empty string, so no need to log empty string